app/crud/crud_test_run_execution.py

```diff
@@ -85,11 +86,24 @@ def get_multi(
             )
 
         if order_by is None:
```

Comment:
Is the suggestion below possible to substitute this **whole** conditional?
What do you think?

```suggestion
            column = self.model.id if order_by is None else getattr(self.model, order_by)
            if sort_order == "desc":
                query = query.order_by(column.desc())
            else:
                query = query.order_by(column.asc())
```

Reply:
Done